lib/config.js

```diff
@@ -0,0 +1,35 @@
+/** @babel */
+
+export const DEFAULT_ACTIVE_FILE_DIR = 'Active file\'s directory';
+export const DEFAULT_PROJECT_ROOT = 'Project root';
+export const DEFAULT_EMPTY = 'Empty';
+
+
+export let config = {
+    caseSensitiveAutoCompletion: {
+        title: 'Case-sensitive auto-completion',
+        type: 'boolean',
+        default: false,
+    },
+    createFileInstantly: {
+        title: 'Create files instantly',
+        description: `When opening files that don't exist, create them
+                      immediately instead of on save.`,
```

Comment:
Yeah. If I did it more often I'd consider a dedent template string processor or something.